_components/banner/guidance/variants/default/implementation.md

```diff
@@ -0,0 +1,6 @@
+- Some `.gov` websites don't represent the federal level of the U.S. government. These sites should adapt the **An official website of the United States government** and **Official websites use .gov** sections to use more accurate language specific to the site.
+- Some `.mil` websites don't belong to an official U.S. Department of Defense organization. These sites should adapt the **Official websites use .mil** section to use more technically accurate language: “A **.mil** website operates under the approval of the U.S. Department of Defense.”
+- The banner should directly follow the `skipnav` component.
+- Set the banner background color with `$theme-banner-background-color`. Banner text will update automatically.
```

Comment:
```suggestion
- Set the banner background color with `$theme-banner-background-color`. Banner text color will update automatically.
```

Reply:
Fixed this in a separate commit. Thank you!